src/control/server/mgmt_pool.go

```diff
@@ -101,41 +102,45 @@ func (svc *mgmtSvc) calculateCreateStorage(req *mgmtpb.PoolCreateReq) error {
 	if len(req.GetRanks()) == 0 {
 		return errors.New("zero ranks in calculateCreateStorage()")
 	}
-	if req.GetScmratio() == 0 {
-		req.Scmratio = DefaultPoolScmRatio
+	if len(req.GetTierratio()) == 0 {
+		req.Tierratio = make([]float64, 2)
+		req.Tierratio[0] = DefaultPoolScmRatio
+		req.Tierratio[1] = DefaultPoolNvmeRatio
```

Comment:
Definitely wary of using these hard-coded array lengths and indexes. If you want to do it this way in order to just get it working first, with the understanding that it will be improved later, I'm OK with that. We shouldn't release with the code like this though, it's too fragile.

Reply:
In this release the engine will understand only two tiers: SCM + NVMe. We did not specify how to generate tier ratios so the translation from default values to ratios will be hardcoded. For this case the result is an array with two entries. If specified differently, this will have to be changed anyway.
Note this will be a valid input in the future, while the tool allows the user to pass an arbitrary number of ratios - in that case the code in question does not execute.
I'm sorry, I don't see how the highlighted code would be too fragile.

However, I do see an issue with the code below and handling of arbitrary number of tiers. Changing that.